docs/developer/DEVELOP.md

```diff
@@ -4,11 +4,18 @@ Instructions on how to setup local dev environment and contribute to osidb.
 
 ## Development environment setup
 
+### Minimal requirements
+
+The following dependencies are required for development and deployment:
+* make
+* podman (docker)
+* podman-compose (docker-compose)
```

Comment:
This is a copypaste from [this section](https://github.com/RedHatProductSecurity/osidb/blob/master/docs/developer/DEVELOP.md#debugging-dev-env). I'd either remove it from there and reference this section instead, or don't add this as it doesn't provide any new info.